### PR TITLE
Add missing kraken, cache-clean & count name

### PR DESCRIPTION
**Pull request by priyesh777**
## Addresses:
- yarn cache issue:  https://github.com/the-deep/client/issues/2330#issue-1106605151 

## Changes:
- yarn cache issue
- count keyword changed to project-activity
- addition of missing information and kraken in multiple places

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,6 +8,6 @@ RUN apk update\
 WORKDIR /code
 
 COPY ./package.json ./yarn.lock /code/
-RUN yarn install --frozen-lockfile
+RUN yarn install && yarn cache clean --frozen-lockfile
```
Comment (tnagorra) [inline Dockerfile]:
```suggestion
RUN yarn install --frozen-lockfile && yarn cache clean
```

File: app/views/Project/EntryEdit/index.tsx [hunk 2]
```diff
@@ -1423,26 +1424,42 @@ function EntryEdit(props: Props) {
...
-                                        direction="vertical"
-                                        spacing="comfortable"
-                                        rendererParams={entryDataRendererParams}
-                                        filtered={false}
-                                        errored={false}
-                                        pending={false}
-                                        emptyIcon={(
-                                            <Kraken
-                                                variant="search"
+                                    {formValue && formValue.entries && formValue.entries?.length > 0
```
Comment (tnagorra) [inline app/views/Project/EntryEdit/index.tsx]:
```suggestion
                                    {(formValue?.entries?.length ?? 0) > 0
```

File: app/views/Home/ProjectItem/index.tsx [hunk 3]
```diff
@@ -349,7 +349,7 @@ function ProjectItem(props: RecentProjectItemProps) {
                                     isAnimationActive={false}
                                 />
                                 <Area
-                                    dataKey="count"
+                                    dataKey="project activity"
```
Comment (tnagorra) [inline app/views/Home/ProjectItem/index.tsx]: Doesn't changing dataKey also change the key selector?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/views/Project/EntryEdit/index.tsx [hunk 4]
```diff
@@ -1423,26 +1424,42 @@ function EntryEdit(props: Props) {
...
-                                        direction="vertical"
-                                        spacing="comfortable"
-                                        rendererParams={entryDataRendererParams}
-                                        filtered={false}
-                                        errored={false}
-                                        pending={false}
-                                        emptyIcon={(
-                                            <Kraken
-                                                variant="search"
+                                    {(formValue?.entries?.length ?? 0) > 0
```
Comment (tnagorra) [inline app/views/Project/EntryEdit/index.tsx]: Doesn't virtualize list view support empty component?

File: app/views/Home/ProjectItem/index.tsx [hunk 5]
```diff
@@ -349,7 +349,7 @@ function ProjectItem(props: RecentProjectItemProps) {
                                     isAnimationActive={false}
                                 />
                                 <Area
-                                    dataKey="count"
+                                    dataKey="activity"
```
Comment (tnagorra) [inline app/views/Home/ProjectItem/index.tsx:353]: We should also be able to actually define the label.
We may need translated text or "Activity".
The current approach is very limited.

File: app/views/Project/EntryEdit/index.tsx (hunk 4 above)
Comment (priyesh777) [reply app/views/Project/EntryEdit/index.tsx]: Message component was implemented as it covers the sufficient container area ans also with the reason that "emptyIcon" component of VirtualizedListView was not supporting the Kraken properly.

File: app/views/AnalyticalFramework/components/Canvas/index.tsx [hunk 6]
```diff
@@ -326,33 +327,47 @@ function Canvas<T>(props: Props<T>) {
...
+                <Message
+                    icon={(
+                        <Kraken
+                            variant="sleep"
+                        />
+                    )}
+                    message="No widgets found."
+                />
+            </div>
+        )
```
Comment (AdityaKhatri) [inline app/views/AnalyticalFramework/components/Canvas/index.tsx]: This is not necessary.

File: app/views/AnalyticalFramework/PrimaryTagging/index.tsx [hunk 7]
```diff
@@ -419,101 +421,116 @@ function PrimaryTaggingInput<K extends string>(props: PrimaryTaggingInput<K>) {
...
-                                    }
-                                />
-                            </TabPanel>
-                        ))
+                        <div className={styles.noWidgetsSelected}>
+                            <Message
+                                icon={
+                                    <Kraken variant="sleep" />
+                                }
+                                message="No widgets found."
```
Comment (tnagorra) [inline app/views/AnalyticalFramework/PrimaryTagging/index.tsx:531]:
```suggestion
                                message="No sections found."
```

Comment (tnagorra) [reply app/views/AnalyticalFramework/PrimaryTagging/index.tsx:531]: We should not hide NonFieldErrors even when there is not section

File: app/views/AnalyticalFramework/components/Canvas/index.tsx (hunk 6 above)
Comment (tnagorra) [inline app/views/AnalyticalFramework/components/Canvas/index.tsx]: 1. We already have ListView to handle empty messages here.
2. Why change emptyMessage on ListView

File: app/views/Home/ProjectItem/index.tsx [hunk 8]
```diff
@@ -349,7 +349,7 @@ function ProjectItem(props: RecentProjectItemProps) {
                                     isAnimationActive={false}
                                 />
                                 <Area
-                                    dataKey="count"
+                                    dataKey="Activity"
```
Comment (tnagorra) [inline app/views/Home/ProjectItem/index.tsx]: We should not change dataKey

File: app/views/Project/EntryEdit/index.tsx [hunk 9]
```diff
@@ -816,11 +817,11 @@ function EntryEdit(props: Props) {
         setSelectedEntry(entryId);
     }, [createRestorePoint]);
 
-    const currentEntryIndex = formValue.entries?.findIndex(
+    const currentEntryIndex = formValue?.entries?.findIndex(
```
Comment (tnagorra) [inline app/views/Project/EntryEdit/index.tsx]: Any reason `?` was added here?

File: app/views/Project/EntryEdit/index.tsx [hunk 10]
```diff
@@ -1589,26 +1590,42 @@ function EntryEdit(props: Props) {
...
+                                                    )}
+                                                    message="No entries found."
+                                                />
+                                            </div>
                                         )}
-                                        emptyMessage="No entries found"
-                                        messageIconShown
-                                        messageShown
-                                    />
+
```
Comment (tnagorra) [inline app/views/Project/EntryEdit/index.tsx:1629]: VirtualizedListView should handle emptyMessage

File: app/views/AnalyticalFramework/Review/index.tsx [hunk 11]
```diff
@@ -37,7 +39,7 @@ function Review(props: Props) {
     const secondaryTagging = secondaryTaggingFromProps;
 
     const [selectedSection, setSelectedSection] = useState<string | undefined>(
-        primaryTagging[0]?.clientId,
+        primaryTagging && primaryTagging[0]?.clientId,
```
Comment (tnagorra) [inline app/views/AnalyticalFramework/Review/index.tsx]: Any reason this was changed?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/views/Project/EntryEdit/index.tsx (hunk 10 above)
Comment (priyesh777) [reply app/views/Project/EntryEdit/index.tsx:1629]: The mentioned issue required that the empty message should appear at the center of the component and to fix this issue a separate component hiding it's layer was used.

File: app/views/AnalyticalFramework/Review/index.tsx (hunk 11 above)
Comment (priyesh777) [reply app/views/AnalyticalFramework/Review/index.tsx]: Deep scan showed the null-check error in that line

File: app/views/AnalyticalFramework/components/Canvas/index.tsx (hunk 6 above)
Comment (priyesh777) [reply app/views/AnalyticalFramework/components/Canvas/index.tsx]: The mentioned issue required that the empty message should appear at the center of the component and to fix this issue a separate component hiding it's layer was used. (ref.1) 
For (ref.2) the emptyMessage has been changed to "No widgets list found."

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

File: app/views/AnalyticalFramework/components/Canvas/index.tsx (hunk 6 above)
Comment (priyesh777) [reply app/views/AnalyticalFramework/components/Canvas/index.tsx]: This has been discussed recently.

**Pull request merged**